spec.html

```diff
@@ -43545,43 +44172,83 @@ <h1>
           ValidateIntegerTypedArray (
             _typedArray_: an ECMAScript language value,
             optional _waitable_: a Boolean,
-          ): either a normal completion containing either an ArrayBuffer or a SharedArrayBuffer, or a throw completion
+          ): either a normal completion containing an Integer-Indexed Object With Buffer Witness Record, or a throw completion
         </h1>
         <dl class="header">
         </dl>
         <emu-alg>
           1. If _waitable_ is not present, set _waitable_ to *false*.
```

Comment:
`waitable` is now always passed and should be made non-optional.

Reply:
Done.

Reply:
@bakkot is it worth trying to get ecmarkup to detect optional AO params that are never omitted?

Reply:
@ljharb See https://github.com/tc39/ecmarkup/issues/486